guides/v2.3/graphql/reference/quote.md

```diff
@@ -27,12 +137,360 @@ mutation {
 
 **Response**
 
-The response is the quote ID, which is sometimes called the cart ID.
+The response is the quote ID, which is sometimes called the cart ID. The remaining examples in this topic will use this cart ID.
 
 ```json
 {
   "data": {
-    "createEmptyCart": "6XZA7q1ooLEI0jLz8DfFrfruEqgxGzlt"
+    "createEmptyCart": "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C"}
   }
 }
 ```
+
+### Add and remove coupons from a cart
```

Comment:
Move the coupons mutations to the bottom. They won't be used as frequently as the others.

Reply:
moved